entrypoint/validatecluster/validatecluster_test.go

```diff
@@ -58,6 +58,10 @@ var _ = Describe("Test:", func() {
 		It("Verifies LoadBalancer Service", func() {
 			testcase.TestServiceLoadBalancer(true)
 		})
+
+		It("Verifies Traefik IngressRoute", func() {
```

Comment:
this test will not be needed the upgrade flows ?

Reply:
Actually it's better to have in the upgrade too yeah. I'll add it